### ISSUE TITLE
Still see extra groups under Alerts some of the time

### ISSUE THREAD

**Issue by davidnuzik** [rancher/rancher#19913]
Version: v2.2.3-rc3, v2.2.3-rc4

**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
- Go to the Alerts page

**Result:**
Somewhat rarely (1/8 chance?) you will still observe the duplicate groups listed.
![image](https://user-images.githubusercontent.com/45179589/56999538-69f3d580-6b64-11e9-93de-40b3b630b33f.png)

**Other details that may be helpful:**
I am having a very difficult time determining the cause. Sometimes if I'm away from the server for 10 or so minutes and go back to global page then back to Alerts I'll see this. I'm not certain how to reproduce this it just happens sometimes with regular use -- just less often as when it was first reported.

I am using incognito chrome with empty cache so I don't run into any strange caching issues, etc.

**Comment by loganhz** [comment]
Reproduce steps:

1. Create cluster A and remove all alert rules but keep alert groups
2. Create cluster B and remove all alert rules and groups
3. Go to cluster A cluster alert page
4. Switch to cluster B

Then you will still see the groups from cluster A

**Comment by loganhz** [comment]
will be available in rc6

**Comment by davidnuzik** [comment]
Version: Master (v2.3) (5/7/19)

Refer to issue for testing in v2.2.3-rc6: #20023 

I tested in master and can no longer reproduce the issue. I followed Logan's suggested steps and also smoke tested for a while to try to reproduce the issue in other ways, such as waiting for a time, having various clusters with different alert rules/groups/etc. I still am not able to run into the issue so I believe this is safe to close. I will continue to keep an eye out to see if the issue happens again.
